Document VTTRegion instance properties (#44494)
mdn/content: files/en-us/web/api/vttregion/regionanchory/index.md

---
title: "VTTRegion: regionAnchorY property"
short-title: regionAnchorY
slug: Web/API/VTTRegion/regionAnchorY
page-type: web-api-instance-property
browser-compat: api.VTTRegion.regionAnchorY
---

{{APIRef("WebVTT")}}

> The **`regionAnchorY`** property of the {{domxref("VTTRegion")}} interface represents the y-coordinate of the region anchor, as a percentage of the region.

Reviewer: ```suggestion
The **`regionAnchorY`** property of the {{domxref("VTTRegion")}} interface represents the y-coordinate of the region anchor, as a percentage of the region's height.
```

Again, good to state this.

## Value

> A number, in the range `0` to `100`, giving the y-coordinate of the region anchor as a percentage of the region. The default is `100`.

Reviewer: ```suggestion
A number, in the range `0` to `100` inclusive, representing the y-coordinate of the region anchor as a percentage of the region's height. The default is `100`.
```

## Examples

> In the following example a new {{domxref("VTTRegion")}} is created, then the value of `regionAnchorY` is set to `70`. The value is then printed to the console.

Reviewer: ```suggestion
In the following example, a new {{domxref("VTTRegion")}} is created, then the value of `regionAnchorY` is set to `70`. The value is then printed to the console.
```
